tools/cal_scripts/ctd_cal_parser.py

```diff
@@ -109,6 +109,7 @@ def read_cal(self, filename):
                     self.serial = '16-'
 
                 if key == 'SERIALNO':
+                    print('sn found')
```

Comment:
either use logging service or remove if it's just a temporary statement.

Reply:
This was a debugging statement and will be removed in the next commit